Tools/Scripts/pgo-profile-summarize

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env python3
+
+import argparse
+import subprocess
+import os
+
+def pad(string, maxLength):
+    if len(string) > maxLength:
+        return string[:maxLength - 1] + u"…"
+    return string.ljust(maxLength)
+
+def shell(command):
+    process = subprocess.Popen(command, shell=True, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+    return process.communicate()[0].strip()
+
+def shortened(name):
+    return name if len(name) < 200 else name[0:99] + u"…" + name[-98:]
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser(description="Dumps function names in a given .profraw file, sorted in descending order by function count")
+    parser.add_argument("file", help="Path to the .profraw file")
+    args = parser.parse_args()
+    if os.path.isfile(args.file):
+        profilePath = args.file
+    else:
+        print("No file at %s" % args.file)
+        exit()
+
+    rawOutput = shell("xcrun -sdk macosx llvm-profdata show --all-functions --value-cutoff=10 %s | c++filt -n" % args.file)
```

Comment:
Ditto above. If you wanted to do bypass `shell=True`, I think you coudl using two separate Popen instances, chained together with `subprocess.Pipe`, but that's probably more trouble than it's worth.